docs/source/contributor-guide/index.md

```diff
@@ -61,13 +61,13 @@ Testing setup:
 
 Formatting instructions:
 
-- [ci/scripts/rust_fmt.sh](../../../ci/scripts/rust_fmt.sh)
-- [ci/scripts/rust_clippy.sh](../../../ci/scripts/rust_clippy.sh)
-- [ci/scripts/rust_toml_fmt.sh](../../../ci/scripts/rust_toml_fmt.sh)
+- [ci/scripts/rust_fmt.sh](https://github.com/apache/arrow-datafusion/blob/master/ci/scripts/rust_fmt.sh)
```

Comment:
This change does mean that they won't work locally on people's machines, but not sure how much of an issue that is... It would be nice if we could have both work

Reply:
I guess I could add a script to copy the files, search and replace the links, then build the HTML  ...

Reply:
@tustvold I changed approach here and added a script to fix up the links when publishing